docs/column.pin.md

```diff
@@ -1,12 +1,25 @@
 # qgrid 
 ## markup
 ```html
-<q-grid rows='data'></q-grid>
+<q-grid rows="$ctrl.rows">
+	<q-grid:columns>
+		<q-grid:column isPinned="true" key="name.last" title="Last Name" value="$ctrl.getLastName($row)"></q-grid:column>
+		<q-grid:column key="birthday" title="Birthday"></q-grid:column>
+		<q-grid:column key="gender" title="Gender"></q-grid:column>
+	</q-grid:columns>
+</q-grid>
 ```
 ## script
 ```javascript
-controller.$inject = ['$scope'];
-function controller($scope){
-	$scope.data = [];
+controller.$inject = ['$scope', 'qgrid'];
+function controller($scope, qgrid){
+  $scope.columns = [
+          {key: 'firstName', label: 'First Name', type: 'text', isVisible: true, isDefault: true, isPinned: true},
```

Comment:
Yes, in JS-configuration we can use is- prefix, it looks good. pin='left|right' looks good too